<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File</class>
 <widget class="QMainWindow" name="File">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#listView
{
border:3px solid rdb(219,193,200);
border-radius:20px;
}
#centralwidget{
background-color: lavender;
}
</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit {
    border-radius: 10px;
 border: 2px solid yellow;
}</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>741</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#listView {
 border: 2px solid yellow;
    border-radius: 50px;
}</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>741</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#tableWidget {
    border: 2px solid yellow; /* Замените &quot;red&quot; на цвет, который вам нравится */
  
}</string>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string>Имя файла</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Размер </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Разрешение</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Глубина цвета</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сжатие</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_3
{
background-color: #FFCC99;
}</string>
     </property>
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <zorder>listView</zorder>
    <zorder>lineEdit</zorder>
    <zorder>tableWidget</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>3</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
